conf/default.toml

```diff
@@ -18,6 +18,26 @@ proxy_path = ""
 # [map]
 # max_zoom = 10
 # max_extent = [-418263.418776, 5251529.591305, 961272.067714, 6706890.609855]
+#
+# Baselayer: The default baselayer can be overridden with three properties:
+#   type: Indicates the baselayer type. Possible values are 'xyz', 'wms', 'wfs'. Mandatory if you configure a baselayer.
+#   url: Baselayer endpoint URL. Mandatory if you configure a baselayer.
+#   name: Baselayer name. Mandatory for 'wms', 'wfs' types where it indicates the layer name.
+# baselayer.type = "xyz"
```

Comment:
I think `baselayer_type` would make more sense, and would be consistent with other keys

Reply:
Ok, I had this reflex, as it will be in the same JS object later, but indeed can adapt this to stay consistent with other configs.